tools/xtask/Cargo.toml

```diff
@@ -11,7 +11,7 @@ walkdir = "2.3.2"
 strum = "0.24.0"
 anyhow = "1.0.56"
 strum_macros = "0.24.0"
-cargo = { version = "0.67.0", features = ["vendored-openssl"] }
+cargo = { version = "0.67.0" }
```

Comment:
So openssl is now "fixed" or does this muck up our work around?  Probably the same question you were asking @PandaZ3D about workflow?